packages/chart/src/components/BarChart.tsx

```diff
@@ -94,6 +97,44 @@ export default function BarChart({ xScale, yScale, seriesScale, xMax, yMax, getX
     return Math.ceil(context.measureText(text).width)
   }
 
+  function cleanData(data) {
+    let cleanedup = []
+    //console.log('## Data to clean=', data)
+    data.forEach(function (d, i) {
+      //console.log("clean", i, " d", d);
+      let cleanedBar = {}
+      Object.keys(d).forEach(function (key) {
+        if (key === 'Date') {
+          // pass thru
+          cleanedBar[key] = d[key]
+        } else {
+          // remove comma and dollar signs
+          let tmp = d[key] != null && d[key] != '' ? d[key].replace(/[,\$]/g, '') : ''
+          if ((tmp !== '' && tmp !== null && !isNaN(tmp)) || (tmp !== '' && tmp !== null && /\d+\.?\d*/.test(tmp))) {
+            cleanedBar[key] = tmp
+          } else {
+            // return nothing to skip bad data point
+          }
+        }
+      })
+      //console.log("cleanedBar=", cleanedBar);
+      cleanedup.push(cleanedBar)
+    })
+    //console.log('## cleanedData =', cleanedup)
+    return cleanedup
+  }
+  const isNumber = value => {
```

Comment:
This could be beneficial to other charts so its possible we want it in `core/helpers` or to lift it up to `CdcChart`

Reply:
Yes I thought of that but if you look at the code there is an if statement related to whatever is not data to be checked like "Date" and that changes.  It's been a couple of days so I forget where but it was some other field in another place that isNumber was used so that's why I did not make it a helper.  Another option is have several versions like isNumberExcludeDate and isNumberExcludeSomeotherfield or perhaps change it to pass in the name of the key to exclude: isNumberExcludeKey

Reply:
Lifted it up to CdcChart